steps/generate_analytical_dataset.py

```diff
@@ -11,13 +11,16 @@
 
 import boto3
 import requests
-
+from datetime import datetime
```

Comment:
this will cause issues with other methods. It is already fixed in the new master. Please try use that one instead.

Reply:
for example datetime.timedelta will now break. 
To use it for example for your strptime method. use: datetime.datetime.strptime